docs/advanced-recipes/custom-useatom-hooks.mdx

```diff
@@ -14,12 +14,16 @@ This page shows the ways of creating different utility functions. Utility functi
 import { useAtom } from 'jotai'
 import { selectAtom } from 'jotai/utils'
 
+/* if an atom is created here, please use `useMemo(() => atom(initValue), [initValue])` instead. */
 export function useSelectAtom(anAtom, keyFn) {
-    return useAtom(selectAtom(anAtom, keyFn));
+    return useAtom(selectAtom(
+        anAtom, 
+        useCallBack(keyFn, [])

```

Comment:
This is not a good example. Please make it realistic, something like:
```js
useCallback((state) => state[prop], [prop])
```

Reply:
How about `prop` with single quote
`useCallback((state) => state['prop'], [])`